packages/svg/src/icons/ZoomOutIcon.tsx

```diff
@@ -0,0 +1,23 @@
+import { FC } from "react";
+import { IconProps } from "./types";
+
+export const ZoomOutIcon: FC<IconProps> = ({
```

Comment:
we are still having inconsistency about which properties are supported by different icons .